src/components/app-container.js

```diff
@@ -57,10 +61,13 @@ const Wrapper = styled.div`
 function clearAllLocalData() {
     localStorage.setItem('userPresets', JSON.stringify({
         "data": []
-    }))
+    }));
     localStorage.setItem('customSpaces', JSON.stringify({
         "data": []
-    }))
+    }));
+    localStorage.setIteam('favoriteSpaces', JSON.stringify({
```

Comment:
misspelled. will cause an error on attempt to clear data

Reply:
Good catch, thank you!